paper-dropdown-menu-light.html

```diff
@@ -288,15 +288,16 @@
       opened="{{opened}}"
       close-on-activate
       allow-outside-scroll="[[allowOutsideScroll]]">
-      <div class="dropdown-trigger">
+      <!-- support hybrid mode: user might be using paper-menu-button 1.x which distributes via <content> -->
```

Comment:
Is this true? You have a slot below so i don't think the class will work? Or does Polymer magically do this for you?